src/BW/Vkontakte.php

```diff
@@ -381,4 +427,11 @@ protected function curl($url)
 
         return $result;
     }
+
+    public function __destruct()
```

Comment:
Could you move this method up (below constructor)? I prefer to declare all magic methods first.